### ISSUE TITLE
Why don't you use 'batchnorm' in your model?

### ISSUE THREAD

**Issue by seekFire** [rongliangzi/Dense-Scale-Network-for-Crowd-Counting#2]

**Comment by rongliangzi** [comment]
In experiments on shanghai tech part A dataset, including batch norm does not show improvement. Up to now I have not gotten theoritical explanation for that.

**Comment by seekFire** [comment]
@rongliangzi Ok, but whether the training process becomes harder without batchnorm?

**Comment by rongliangzi** [comment]
the loss reduce normally in training process.

**Comment by seekFire** [comment]
@rongliangzi, OK, Thank you very much!
